caching-service-package/src/main/resources/bin/start.sh

```diff
@@ -273,8 +283,6 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${CACHING_CODE} java \
   -Djgroups.bind.port=${ZWE_configs_storage_infinispan_jgroups_port:-7098} \
   -Djgroups.keyExchange.port=${ZWE_configs_storage_infinispan_jgroups_keyExchange_port:-7118} \
   -Djgroups.tcp.diag.enabled=${ZWE_configs_storage_infinispan_jgroups_tcp_diag_enabled:-false} \
```

Comment:
how is the location set now? based on the workspace? Where can I see the directory that will be used?
Also, btw, does it still need to be in a non-shared filesystem?